osd/user-ca-bundle_invalid.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Misconfigured additional Trusted CA bundle",
+    "description": "Your cluster requires you to take action because the Additional Trusted CA Bundle that you have applied to your cluster is corrupt or cannot be correctly parsed. This may impact your cluster's ability to communicate with any external resources that require that bundle, including those accessible via a cluster-wide proxy. Please remove or update the Additional Trusted CA Bundle to ensure it is in a parseable, plaintext PEM-formatted format. If you require any further assistance, please file a support request.",
```

Comment:
Should we reference https://docs.openshift.com/container-platform/4.9/networking/enable-cluster-wide-proxy.html#enable-cluster-wide-proxy ? or not due to this being configured via OCM ?

Reply:
Should not point them the OCP doc since the proxy is not configured the same way with OCP.

Reply:
I agree with @bmeng , the OCP docs refer to permissions that ultimately a customer will not be allowed to do. Once our proxy docs are added to the OSD/ROSA documentation there will be a URL we can point them at.